ntbs-service/Pages/Shared/_Header.cshtml

```diff
@@ -37,6 +32,23 @@
             </div>
         </div>
     </div>
+
+    @if (!environmentDescription.ContainsLiveData)
+    {
+        <div class="govuk-phase-banner live-data-warning-banner">
+            <div class="govuk-width-container">
+                <p class="govuk-phase-banner__content ">
+                    <strong class="govuk-tag govuk-phase-banner__content__tag">
+                        @environmentDescription.DisplayName
+                    </strong>
+                    <span class="govuk-phase-banner__text">
+                        Please do not enter live data.
```

Comment:
I wonder if we could make this message a little more informative... e.g. "This site is intended to be used for testing purposes only. Please do not enter live data."

That still doesn't sound perfect to me, but perhaps it's a little better. What do you think?

Reply:
This is what it would look like with that message. I think the main thing I would be worried about is it being too long, so users are more tempted to gloss over it, rather than the very short and to the point "Please do not enter live data", which does assume the user knows the differences between the training/live systems, and just reminds them of the most important thing (to not enter live data). The longer message explains the purpose of the site (which hopefully our users should know by this point anyway), where the shorter one just serves as a reminder. What do you think?

![longer_banner](https://user-images.githubusercontent.com/3650110/118096327-1f4efa00-b3c9-11eb-9152-a8ad2da3cce9.png)

Reply:
I think it's worth asking @aptemus to weigh in on this

Reply:
I do think it's worth stressing the data in the system is synthetic data (feels like a better word than fictional?).  But I like @JoshCadney 's suggestion.